BookingsApi/BookingsApi/Controllers/WorkHoursController.cs

```diff
@@ -169,7 +171,8 @@ public async Task<IActionResult> UpdateVhoNonAvailabilityHours(string username,
                 return NotFound();
             }
             
-            var hourValidationResult = new UpdateNonWorkingHoursRequestValidation().ValidateHours(request, existingHours);
+            var nonDeletedHours = existingHours.Where(x => !x.Deleted).ToList();
```

Comment:
is this definitely needed? The GetVhoNonAvailableWorkHoursQuery already filters out deleted hours:

`return justiceUser.VhoNonAvailability.Where(x => x.Deleted != true).ToList() ?? new List<VhoNonAvailability>();`